src/CodeFormatter/Program.cs

```diff
@@ -117,11 +131,42 @@ private static int RunFormatCommand(FormatOptions options)
 
                 return FAILED;
             }
+            finally
+            {
+                stopwatch.Stop();
+                Console.WriteLine("Total time: {0}", stopwatch.Elapsed);
+            }
         }
 
-        private static async Task<int> RunFormatAsync(FormatOptions options, CancellationToken cancellationToken)
+        private static IAnalyzerAssemblyLoader _analyzerAssemblyLoader;
+        private static IAnalyzerAssemblyLoader AnalyzerAssemblyLoader {
+            get
+            {
+                if(_analyzerAssemblyLoader == null)
+                {
+                    // use Roslyn's existing simple loader, there're no special requirements for our usage
+                    var loaderAssembly = Assembly.Load((typeof(CommandLineProject)).Assembly.FullName);
+                    _analyzerAssemblyLoader = (IAnalyzerAssemblyLoader)Activator.CreateInstance(loaderAssembly.GetType("Microsoft.CodeAnalysis.SimpleAnalyzerAssemblyLoader"));
```

Comment:
Do you really need Roslyn's assembly loader? That does a bunch of caching and hooks assembly resolve to be able to find dependencies from any of the analyzer folders. You could impose a simpler model here and have assembly loader simply do a LoadFrom(..) and just require all dependencies to be in the same folder. That would avoid reflection here as well.